client/src/components/paginationTable/PaginationTable.module.css

```diff
@@ -1,5 +1,5 @@
 table {
-    width: 70%;
+    width: 90%;
```

Comment:
Well by doing this you expanded the width of all pagination tables in the website..
Since `70%` looks quite good in Tesel's search users, following & followers, what I suggest you to do is to add "tableOverrideStyle" to PaginationTable's props (default = `{}`), then in `SearchSeries` component add to `PaginationTable`:
`tableOverrideStyle={{ width: "90%" }}`
Will make everybody happy :)